# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/prusa-slicer/wxwidget-version.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- PrusaSlicer-version_2.6.0-alpha6/src/CMakeLists.txt.vanilla	2023-04-15 23:51:04.163763870 +0200
+++ PrusaSlicer-version_2.6.0-alpha6/src/CMakeLists.txt	2023-04-15 23:52:06.462763836 +0200
@@ -52,10 +52,13 @@
         if (SLIC3R_WX_STABLE)
             find_package(wxWidgets 3.0 REQUIRED COMPONENTS base core adv html gl)
         else ()
-            find_package(wxWidgets 3.1 QUIET COMPONENTS base core adv html gl)
+            #find_package(wxWidgets 3.1 QUIET COMPONENTS base core adv html gl)
+            message(STATUS "Trying to find wxWidgets in CONFIG mode...")
+            find_package(wxWidgets 3.1)
+            slic3r_remap_configs(wx::wxhtml wx::wxadv wx::wxgl wx::wxcore wx::wxbase RelWithDebInfo Release)
 
             if (NOT wxWidgets_FOUND)
-                message(FATAL_ERROR "\nCould not find wxWidgets 3.1.\n"
+                message(FATAL_ERROR "\nCould not find wxWidgets 3.1.\n"
                     "Hint: On Linux you can set -DSLIC3R_WX_STABLE=1 to use wxWidgets 3.0\n")
             endif ()
         endif ()
